cli/cli_app.go

```diff
@@ -716,7 +725,7 @@ func destroyAll(command string, terragruntOptions *options.TerragruntOptions) er
 	}
 
 	if shouldDestroyAll {
-		return stack.RunAll([]string{command, "-force", "-input=false"}, terragruntOptions, configstack.ReverseOrder)
+		return stack.RunAll([]string{command, "-auto-approve", "-input=false"}, terragruntOptions, configstack.ReverseOrder)
```

Comment:
`-force` is no longer supported on destroy operation. Since the destroy operation is not applied through apply, terraform changed the parameter name to use the same as apply i.e. `-auto-approve`.